code/_onclick/hud/hud.dm

```diff
@@ -205,6 +205,61 @@ datum/hud/New(mob/owner)
 	else if(isguardian(mymob))
 		guardian_hud()
 
+	create_parallax()
+
+
+/client/var/list/obj/screen/spessbg/spessbg = list()
+
+/obj/screen/spessbg
+	var/offset_x = 0
+	var/offset_y = 0
+
+/datum/hud/proc/create_parallax()
+	var/client/C = mymob.client
+	for(var/area/imgarea in all_areas)
+		var/image/img = imgarea.white_overlay
+		if (C.prefs.space_parallax)
+			C.images |= img
+		else
+			C.images -= img
+	if (C.spessbg.len)
+		for(var/obj/screen/spessbg/bgobj in C.spessbg)
+			bgobj.layer = (C.prefs.space_parallax) ? AREA_LAYER + 0.5 : 0
+			C.screen |= bgobj
+		return
+	for(var/i=0;i<4;i++)
+		var/obj/screen/spessbg/bgobj = new /obj/screen/spessbg()
+		if(i & 1)
+			bgobj.offset_x = 480
+		if(i & 2)
+			bgobj.offset_y = 480
+		bgobj.icon = 'icons/mob/screen1_full.dmi'
+		bgobj.icon_state = "spess"
+		bgobj.name = "spess"
+		bgobj.screen_loc = "CENTER-7:[bgobj.offset_x],CENTER-7:[bgobj.offset_y]"
+		bgobj.layer = (C.prefs.space_parallax) ? AREA_LAYER + 0.5 : 0
+		bgobj.blend_mode = BLEND_MULTIPLY
+		bgobj.mouse_opacity = 0
+		C.spessbg += bgobj
+		C.screen += bgobj
+	update_parallax()
+/datum/hud/proc/update_parallax()
+	var/atom/posobj = mymob
+	while(!istype(posobj,/turf))
+		posobj = posobj.loc
+	for(var/obj/screen/spessbg/bgobj in mymob.client.spessbg)
+		bgobj.screen_loc = "CENTER-7:[bgobj.offset_x-posobj.x],CENTER-7:[bgobj.offset_y-posobj.y]"
+
+/atom/movable/Move()
```

Comment:
In general, it's not great to make more than one version of a proc; It makes it so that the return value can be very iffy. Also, using an `istype(src, /mob)` in /atom/movable/Move() is not good.

You should move the contents code to [here](https://github.com/ParadiseSS13/Paradise/blob/master/code/game/atoms_movable.dm#L90), and split off the `istype(src, /mob)`  into a new /mob/Move() (yes i do realize I just said making more versions of a proc is bad, and it is, but mob/Move() has no central location, so it's already fucked. You just **have** to make sure that you use `. = ..()` inside of it.